FaunaDB.Client/Client/DefaultClientIO.cs

```diff
@@ -105,5 +98,17 @@ static string QueryString(IReadOnlyDictionary<string, string> query)
                 q[kv.Key] = kv.Value;
             return q.ToString();
         }
+
+        static HttpClient CreateClient(Uri endpoint, TimeSpan timeout)
+        {
+            var client = new HttpClient();
+            client.BaseAddress = endpoint;
+            client.Timeout = timeout;
+            client.DefaultRequestHeaders.AcceptEncoding.Add(new StringWithQualityHeaderValue("gzip"));
+            client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
+            client.DefaultRequestHeaders.Add("X-FaunaDB-API-Version", "2.1");
```

Comment:
Just as a note: I think we should create a branch of master like `release-1.0` and add the same line pinning the api version to 2.0.